formats/HtmlFormat.php

```diff
@@ -58,7 +58,7 @@ public function stringify(){
 			$entryContent = '';
 			if($item->getContent()) {
 				$entryContent = '<div class="content">'
-				. $this->sanitizeHtml($item->getContent())
+				. $this->sanitizeHtml((string) $item->getContent())
```

Comment:
this cast is redundant

Reply:
Since you originated this PR can you fix this then?